lib/src/markdown_content.dart

```diff
@@ -52,7 +60,8 @@ List<T> _unique<T>(Iterable<T> l) => l.toSet().toList();
 
 /// Scans a markdown file and extracts its content.
 Future<ExctractedMarkdownContent> scanMarkdownFileContent(File file) async {
-  final text = await file.readAsString();
+  final bytes = await file.readAsBytes();
+  final text = utf8.decode(bytes, allowMalformed: true);
```

Comment:
A better way to do this is to do:
```dart
try {
  text = utf8.decode(bytes, allowMalformed: false);
} on FormatException {
  text = utf8.decode(bytes, allowMalformed: true);
  isMalformedUtf8 = true;
}
```

That way we're not grepping for `_replacementUtf8Char` which the original markdown file could contain... Imagine a README file explaining how UTF8 decoding works :D

Reply:
Done. PTAL.